### ISSUE TITLE
Mobile to web video chat

### ISSUE THREAD

**Issue by nileshlets** [nttcom/peerjs#48]
hello I am trying to connect from iOS to web socket using peer js.
This works for only web to web, while setting to iOS to web or web to iOS peer connections is failed.
Is is possible via this script to make connection between web and iOS and make video call?

**Comment by iwashi** [comment]
Yes, the application with our iOS SDK and the application with our JavaScript library can communicate each other.

**Comment by nileshlets** [comment]
Can you provide example or repo link for same?
Is it paid service ?

**Comment by iwashi** [comment]
> Can you provide example or repo link for same?

Here is for iOS: 
https://github.com/nttcom/SkyWay-iOS-Sample

Please check this document as well:
http://nttcom.github.io/skyway/en/info-mobile.html

> Is it paid service ?

No, our service is for trial now so you can use it for free.
